lib/QC.groovy

```diff
@@ -59,14 +59,14 @@ class QC {
 // Get GATK version
   static def getVersionGATK() {
     """
-    echo "GATK version"\$(java -jar \$GATK_HOME/GenomeAnalysisTK.jar --version 2>&1) > v_gatk.txt
+		gatk-launch ApplyBQSR --help 2>&1| awk -F/ '/java/{for(i=1;i<=NF;i++){if(\$i~/gatk4/){sub("gatk4-","",\$i);print \$i>"v_gatk.txt"}}}'
```

Comment:
I think we can work out the regex in the Python script instead of doing it here, it'll make more sense

Reply:
the gatk-launch is GATK-provided, I do not want to fiddle with that. OTOH it would be nice if they would have a --version option :/

Reply:
I'll look more if there's something similar with the new GATK

Reply:
My feeling is that it is still the easiest way to have the version :/

Reply:
I was thinking more of removing the awk part, and do the regex in the [python script](https://github.com/SciLifeLab/Sarek/blob/master/bin/scrape_tool_versions.py)

Reply:
I see, sure, we can refactor it for the rest of the software later also.